lib/msal-common/src/cache/entities/AccountEntity.ts

```diff
@@ -172,14 +172,14 @@ export class AccountEntity {
      * @param authority
      * @param idToken
      */
-    static createADFSAccount(
+    static createGenericAccount(
```

Comment:
`createAccount` vs `createGenericAccount` - names may not be conveying what we are doing here. Can we change the comments to reflect - `Building non AAD accounts (ADFS or otherwise?)

Reply:
I changed both of the comments on these functions a bit to clarify, good callout.